assets/dev/js/editor/elements/views/base.js

```diff
@@ -659,6 +659,31 @@ BaseElementView = BaseContainer.extend( {
 		this.renderHTML();
 	},
 
+	isAtomicDynamic( dataBinding ) {
+		return !! ( dataBinding.el.hasAttribute( 'data-binding-dynamic' ) && elementorCommon.config.experimentalFeatures.e_nested_atomic_repeaters );
+	},
+
+	isTitle( changedControl ) {
+		const titleSettings = [ 'item_title', 'tab_title' ];
```

Comment:
A clarification question: are we bound to these titles for all nested widgets? 
Is there any way to make these parameters more dynamic?

Reply:
We are not, Nested Tabs title is `tab_title`
And Nested Accordion and Mega Menu titles are `item_title`

We can pass them from the config, but it is more or less the same. 

WDYT?

Reply:
Done